apps/platform-app/src/app/[locale]/(profile)/(components)/CVuploader/CvUploaderForm.tsx

```diff
@@ -8,21 +8,37 @@ import { useTranslations } from 'next-intl'
 import { useState } from 'react'
 import styles from './CvUploaderForm.module.scss'
 
-export function CVuploaderForm() {
+interface CvUploaderFormProps {
+  initialCvUrl: string | null
+}
+
+export function CVuploaderForm({ initialCvUrl }: CvUploaderFormProps) {
   const t = useTranslations(I18nNamespaces.Buttons)
+  const tt = useTranslations(I18nNamespaces.PersonalInfo)
   const [isUploading, setIsUploading] = useState(false)
 
```

Comment:
**nit:** we can extract form value by using useFormixContext, and extract inital url from there without need of passing down this prop that many levels:

```ts
 const { values } =
    useFormikContext<CreateProfileFormValues>()
```